### PR TITLE
Examples: material.shading ==> material.flatShading

### PR DESCRIPTION
**Pull request by WestLangley**
Whew!

It remains to update the loaders.

### CODE REVIEW

File: examples/webvr_rollercoaster.html [hunk 1]
```diff
@@ -165,7 +165,8 @@
 
 			var geometry = new THREE.CylinderBufferGeometry( 10, 10, 5, 15 );
 			var material = new THREE.MeshLambertMaterial( {
-				color: 0xff8080, shading: THREE.FlatShading
+				color: 0xff8080,
+				//flatShading: true // Lambert does not support flat shading
```
Comment (WestLangley) [inline examples/webvr_rollercoaster.html]: @mrdoob Note this. Either use `MeshPhongMaterial`, or use `Geometry` and call `geometry.computeFlatVertexNormals();`.

Comment (mrdoob) [inline examples/webvr_rollercoaster.html]: 👌

**Comment by mrdoob** [comment]
Thanks!

**Comment by Mugen87** [comment]
> It remains to update the loaders.

The material and geometry browsers in the documentation should also be checked.

**Comment by WestLangley** [comment]
>The material and geometry browsers in the documentation should also be checked.

I made those changes in #11724 when I updated the docs.

**Pull request merged**